plugins/modules/ldap_attrs.py

```diff
@@ -44,7 +44,7 @@
   check_mode:
     support: full
   diff_mode:
-    support: none
+    support: full
```

Comment:
```suggestion
    support: full
    version_added: 8.5.0
```

Reply:
@felixfontein 
Isn't this something that could/should be replace by the doc fragment ?

Reply:
No, `support` needs to be specified per module, and `version_added` needs to be added if `support` in increased after the module was added. This is nothing the doc fragment can really take over (except for special classes of modules, like `_facts` and `_info` modules, where `support` must have a specific value).